CHANGES.md

```diff
@@ -20,7 +20,7 @@ Developer Tools:
 
 Themes:
 
-- Nothing yet.
+- Added `Rosé Pine` theme [William Wilkinson][]
```

Comment:
Need to add a link as well line 30, etc.

Reply:
Oops 😂 , done.